### PR TITLE
Internal Affairs Agents now glow slightly

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Doctor Pavlos? I'm IAA.
IAA
This is a serious PR.
IAAs will glow innately and slightly brighter during April Fools.

:cl:
 * tweak: Nanotrasen's Internal Affairs Agents are now provided with an object that allows them to identify each other in lower-light conditions, which is attached to the jumpsuit by default.

### CODE REVIEW

**Comment by BomberBro** [comment]
Amazing april fools PR, meme PR otherwise

File: code/datums/outfit/civilian.dm [hunk 1]
```diff
@@ -669,6 +669,7 @@
 /datum/outfit/iaa/post_equip(var/mob/living/carbon/human/H)
 	..()
 	H.put_in_hands(new /obj/item/weapon/storage/briefcase/centcomm(H))
+	H.set_light(1, 3, "#006400") //Green
```
Comment (Kurfursten) [inline code/datums/outfit/civilian.dm]: Why not just do this with a glowing badge accessory instead?

**Comment by ghost** [comment]
This should be a gene or something with light level based on the block value.

File: code/datums/outfit/civilian.dm (hunk 1 above)
Comment (boy2mantwicethefam) [reply code/datums/outfit/civilian.dm]: It's funnier without reliance to accessories

**Comment by boy2mantwicethefam** [comment]
Lawyers and Bridge Officers, who share the same outfit datum as the IAAs will no longer share the glowing aspect.
Thanks @ShiftyRail !

**Comment by Eneocho** [comment]
If I take off that mask, would you die?

**Comment by cloutcode** [comment]
add a new feature where everytime a lawyer dies you can actually see them being sent directly to hell

**Comment by Eneocho** [comment]
> 
> 
> add a new feature where everytime a lawyer dies you can actually see them being sent directly to hell

THIS.
Would also make sense IC, since you can spawn on hell as a lawyer on lamprey.

**Comment by BomberBro** [comment]
On further thought I don't like this PR if there's no way for the IAA to lose his shine because it gives a "meta" way to prevent anyone from impersonating an IAA or an IAA antagonist from being able to disguise properly (though the latter is funny).

**Comment by boy2mantwicethefam** [comment]
That's....actually a good point @BomberBro

**Comment by Eneocho** [comment]
> 
> 
> That's....actually a good point @BomberBro

Make the glow a genetic skill and give it to IAAs, like the SOBER trait for detectives or the NOIR trait for gumshoes. It'd be a good middleground between the hilarity of them naturally glowing and the incapacity to impersonate (Ah, I just got cloned and ryetalin'd so no more glow).

**Comment by help-maint** [comment]
The glowing accessory is the best way to do this one IMO

**Comment by Blithering** [comment]
We have glowsticks, could just spawn those in their pockets.

**Comment by Kurfursten** [comment]
Reskin the glowstick so it's a glowie badge

**Comment by boy2mantwicethefam** [comment]
Kurfurst-sama...I kneel....

**Comment by system86** [comment]
> 
> 
> add a new feature where everytime a lawyer dies you can actually see them being sent directly to hell

copypaste goonecode that implements banishment to hell
job dun

**Pull request merged**